library/x509write_csr.c

```diff
@@ -122,68 +130,74 @@ int mbedtls_x509write_csr_set_ns_cert_type( mbedtls_x509write_csr *ctx,
     return( 0 );
 }
 
-int mbedtls_x509write_csr_der( mbedtls_x509write_csr *ctx, unsigned char *buf, size_t size,
-                       int (*f_rng)(void *, unsigned char *, size_t),
-                       void *p_rng )
+static int mbedtls_x509write_csr_der_internal( mbedtls_x509write_csr *ctx,
```

Comment:
Since this is a static internal function, please remove the `mbedtls_` prefix

Reply:
Ok, I will remove the prefix.